code/controllers/voting.dm

```diff
@@ -1,9 +1,15 @@
 var/datum/controller/vote/vote = new()
 
-var/global/list/round_voters = list() //Keeps track of the individuals voting for a given round, for use in forcedrafting.
+var/global/list/round_voters = list() // Keeps track of the individuals voting for a given round, for use in forcedrafting.
 
-datum/controller/vote
-	var/initiator = null
+#define VOTE_RESTART "restart"
+#define VOTE_GAMEMODE "gamemode"
+#define VOTE_CREW_TRANSFER "crew_transfer"
+#define VOTE_ADD_ANTAGONIST "add_antagonist"
+#define VOTE_CUSTOM "custom"
+

```

Comment:
What the point of the these defines, can you add commits on how they are used, and if able where they are being used, what are they replacing. That way later on down the road we know what the hell is actually going on.